src/ServerTelemetryChannel/Implementation/TelemetryChannelEventSource.cs

```diff
@@ -481,6 +482,12 @@ public void ItemRejectedNoInstrumentationKey(string item, string appDomainName =
             this.WriteEvent(67, item ?? string.Empty, this.ApplicationName);
         }
 
+        [Event(68, Message = "Failed to set access permissions on storage directory {0}. Error : {1}.", Level = EventLevel.Warning)]
```

Comment:
Should it direct user to the creation of the custom folder for the telemetry storage in case of non-Windows platforms?

Reply:
This is a warning message and won't be automatically picked up y SDK diagnostics. There is an error message which will inform the user to create folder themselves.